packages/components/src/focal-point-picker/index.js

```diff
@@ -190,8 +190,9 @@ export class FocalPointPicker extends Component {
 			{ x: event.pageX, y: event.pageY },
 			event.shiftKey
 		);
-		this.updateValue( value );
-		this.props.onDrag?.( value, event );
+		const resolvedValue = this.props.resolvePoint?.( value ) ?? value;
+		this.updateValue( resolvedValue );
```

Comment:
Like @stokesman suggested earlier, should we call `resolvePoint` inside `updateValue` instead?